docs/use-cases/exchange.md

```diff
@@ -0,0 +1,138 @@
+    + [2.1.7 Deposit Funds to the OMG Network](#217-deposit-funds-to-the-omg-network)
+    + [2.1.8 Rebalance Wallets](#218-rebalance-wallets)
+  * [2.2 Integration UI](#22-integration-ui)
+
+## 1. Introduction
+
+### 1.1 Overview
+
+The main goal of this document is to demonstrate how a stablecoin issuer or an exchange can add an OMG Network as one of the options to deposit and withdraw a particular ERC20 token (e.g. USDt).
+
+The integration with the OMG Network can significantly help the entire Ethereum ecosystem to offload a certain volume from the rootchain, as well as introduce faster and cheaper transactions with the same security guarantees as to Ethereum network.
+
+### 1.2 Requirements
+
+#### User Personas
+
+1. Customers - participants and active users/traders of a given exchange.
+2. Integrator - a development and management team within an exchange that evaluates the requirements and delivers the integration with the OMG Network.
+3. OMG Network - a technical and management team within the OMG Network that supports the Integrator during the integration process.
+
+#### User Stories
+
+1. As a customer, I can deposit ERC20 tokens (e.g. USDt) from the OMG Network wallet (web or mobile) to my exchange account.
+2. As a customer, I can use deposited funds to make trades on a given exchange.
+3. As a customer, I can withdraw ERC20 tokens from my exchange account into the OMG Network wallet (web or mobile).
+
+## 2. Proposed Design
+
+The proposed design includes improvements to an existing exchange platform. However, there are no strict requirements one has to follow due to the distinct nature of given exchange infrastructure, development approach, UI/UX design, etc.
+
+### 2.1 Integration Scope
+
+The scope for integrating with the OMG Network by a given exchange includes the following steps:
+
+#### 2.1.1 Generate ETH Addresses
+
+As an exchange, you need to be able to generate new Ethereum addresses for your customers. The OMG Network reflects the same addresses as the Ethereum network, thus you don't need to implement extra logic for your platform's wallet. If you already have Ethereum supported, you shouldn't have any issues.
+
+Otherwise, you can create a new public/private key pair using [`web3.eth.accounts.create`](https://web3js.readthedocs.io/en/v1.2.0/web3-eth-accounts.html#create) function provided by the web3 library. 
+
+> This document doesn't cover best practices for managing and storing keys due to a variety of methods different exchanges use to achieve a high level of security guarantees on their platform.
+
+#### 2.1.2 Query Blockchain
+
+It's important to understand how to query data from the OMG Network. There are several things that might be important for any exchange:
+- Transaction details - use `getTransaction(id)` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+- Block details - use `getTransactions(filters)` function by the [`omg-js`](https://docs.omg.network/omg-js/)
```

Comment:
this is not block details exactly

maybe get transaction detail(s) based on Address, or Block number?